libraries-stream/src/test/java/com/baeldung/parallel_collectors/ParallelCollectorsVirtualThreadsManualTest.java

```diff
@@ -0,0 +1,69 @@
+package com.baeldung.parallel_collectors;
+
+import com.pivovarit.collectors.ParallelCollectors;
+import org.junit.Test;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.List;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.function.Supplier;
+import java.util.stream.Stream;
+
+import static java.util.stream.Collectors.toList;
+
+public class ParallelCollectorsVirtualThreadsManualTest {
```

Comment:
The `Manual` effectively excludes it - since those tests are creating unreasonable number of threads (both os and virtual), it's better to not stress tests the Jenkins worker